clonelists/Sony - PlayStation 2.json

```diff
@@ -2101,6 +2111,9 @@
 			["Nickelodeon Rugrats - Rescate Real", 1],
 			["Nickelodeon Les Razmoket - La Rancon Royale", 1]
 		],
+		"Nickelodeon SpongeBob and Friends - Attack of the Toybots": [
```

Comment:
Why did you flip these around and change the priorities?

Reply:
Inverted "Nickelodeon SpongeBob and Friends - Attack of the Toybots" and "Nicktoons - Attack of the Toybots" because the former is Multi6.

Reply:
Ah the clonelists don't work like that. Except for `0`, priority is only relevant in the same region. You'll get the exact same result as you did with the original clonelist.

Reply:
I know... and dunno why I did it like that o_O

Reply:
All good :) It's still a regional equivalent, so should be a `1`, not a `0`.

There's a lot of automated stuff Retool does with languages and regions that the `0` can mess with, and I'm thinking forward to v2 functionality as well.